rasa/nlu/training_data/formats/luis.py

```diff
@@ -1,5 +1,6 @@
 import logging
 import typing
+import warnings
```

Comment:
Why import this lib?
I guess the usage of it is similar to `logging`. :thinking: 
Is there anything that justifies this new import on this file that is not possible with the others imports like `logging`?